app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/FormApproveOrReject.tsx

```diff
@@ -7,38 +7,43 @@ import React from "react";
 import Alert from "react-bootstrap/Alert";
 import Button from "react-bootstrap/Button";
 import Form from "react-bootstrap/Form";
+import { NimbusChangeLogType } from "./temp-types";
 
-const FormApproveOrRejectLaunch = ({
-  rsRequestTimedOut,
-  launchRequestedByUsername,
+const FormApproveOrReject = ({
+  actionDescription,
+  timeoutEvent,
+  reviewRequestEvent,
   isLoading,
   onApprove,
   onReject,
 }: {
-  rsRequestTimedOut?: boolean;
-  launchRequestedByUsername: string;
+  actionDescription: string;
+  timeoutEvent?: NimbusChangeLogType;
+  reviewRequestEvent?: NimbusChangeLogType;
   isLoading: boolean;
   onApprove: () => void;
   onReject: () => void;
 }) => {
+  const ucActionDescription =
+    actionDescription.charAt(0).toUpperCase() + actionDescription.slice(1);
   return (
     <>
-      {rsRequestTimedOut && (
+      {timeoutEvent && (
         <Alert variant="danger">
           <p className="mb-0">
             <span role="img" aria-label="red X emoji">
               ❌
             </span>{" "}
-            Remote Settings request has timed out, please approve and launch
-            through Remote Settings again
+            Remote Settings request has timed out, please approve through Remote
```

Comment:
I think this could be `...please approve and ${actionDescription} through Remote Settings again` to match Figma.

Reply:
Yeah, we might want to verify the language there: In Remote Settings, it's about approving the change to the deployed data - whether that change happens to add, change, or remove an experiment. I don't think Remote Settings mentions anything about our `${actionDescription}` on Experimenter.